contacts/views.py

```diff
@@ -14,3 +14,8 @@ class ContactDetailView(generic.DetailView):
 class AddContactView(generic.CreateView):
     model = Contact
     fields = ['name', 'email', 'phone_number']
+
+
+class AddContactAddressView(generic.CreateView):
+    model = Address
+    fields = ['contact', 'address_selection', 'address_line', 'city', 'state', 'zip_code', 'country']
```

Comment:
i'm not sure how it works but what fields does it show against 'contact'? how do we select contact for the address being added?

Reply:
Contact displays a drop down with all contacts in it (as in admin). You can select the contact and add address against it.

Reply:
dropdown would fail for large data loading...but let it go as-is for now! will see the workaround later!